test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-02T113344.698028Z.results.json

```diff
@@ -8,7 +8,7 @@
             "entity_id": "aaa96233-bf27-44c7-82df-b4dc15ad4d9d",
             "source": {
                 "id": "4b737739-4dc9-5d4b-9989-a4942047c91c",
-                "name": "test"
+                "spec": "test"
```

Comment:
Can you post some evidence that in a real deployment's index, one that's populated from a TDR source, this is the full spec and not just the name?

Reply:
I selected the dev deployment and used the kibana proxy to show the value in the dcp2 bundles index.
Hopefully this is sufficient, bottom of the image:

<img width="1331" alt="Screen Shot 2021-05-20 at 4 25 22 PM" src="https://user-images.githubusercontent.com/1947534/119060941-11304900-b988-11eb-842e-f467a2a62de4.png">